client/package.json

```diff
@@ -71,6 +71,7 @@
     "intl": "^1.2.5",
     "jquery-ui": "^1.12.1",
     "json-loader": "^0.5.4",
+    "lodash": "^4.17.4",
```

Comment:
There are only a few uses of lodash in your PR. Is it possible to rewrite those to not use lodash?